### ISSUE TITLE
AccessControl for publicKey doesn't restrict to biometryCurrentSet flag

### ISSUE THREAD

**Issue by romanbaitaliuk** [agens-no/EllipticCurveKeyPair#52]
Hi,

I've found your helper is really useful. Despite, I couldn't understand the AccessControl for publicKey. When I set it to [.biometryCurrentSet, .privateKeyUsage] it doesn't ask me to authenticate with FaceID/TouchID when encrypting data.

Do you know why is it happening?

I believe there is no point to have those flags when create KeyPair manager, we can just provide AccessControl for privateKey.

Thanks,
Roman

**Comment by hfossli** [comment]
Simulator or device?

**Comment by romanbaitaliuk** [comment]
I've tried on real device.

**Comment by hfossli** [comment]
I agree with your point though I believed it would be the same for public key
